app/views/providers/uploaded_evidence_collections/show.html.erb

```diff
@@ -25,7 +25,7 @@
   <%= render partial: "upload_evidence" %>
 
   <%= form_with(
-        model: @submission_form,
+        model: @submission_form.presence || false,
```

Comment:
Does this only affect file upload forms?

Reply:
Yes. in certain circumstances the page is re-rendered, purposefully without a submission form.
There is a deprecation warning that Rails 8 will fail if that occurs.  It will accept false, but not nil.
This allows the tests to run without the warning and is acceptable to rails 8